src/python/WMCore/WMSpec/Steps/Executors/LogCollect.py

```diff
@@ -176,6 +176,7 @@ def execute(self, emulator=None):
                 logging.error("Indefinite hang during edmCopyUtil/stageIn of logArchives")
             except StageOutFailure:
                 logging.error("Unable to stageIn logArchives")
+                raise
```

Comment:
@amaltaro, Should we raise here or ignore? raise means if we fail to get the one of input logArchive fail the logCollect job. If we ignore we can still have logCollectJob successful with missing logArchive file (some or all).

Reply:
We can ignore it. LogCollect is supposed to fail *only* when we fail to collect ALL jobs tarball. If it succeeds collecting at least one, then the job is supposed to succeed.